### ISSUE TITLE
Ink/Stitch experienced an unexpected error. This means it is a bug in Ink/Stitch.

### ISSUE THREAD

**Issue by Jigfaber** [inkstitch/inkstitch#1989]
Ink/Stitch Version: v2.2.0 (windows)

Traceback (most recent call last):
  File "inkstitch.py", line 84, in <module>
  File "inkex\base.py", line 231, in run
  File "lib\extensions\troubleshoot.py", line 29, in effect
  File "lib\elements\satin_column.py", line 455, in validation_errors
  File "lib\elements\satin_column.py", line 954, in to_stitch_groups
  File "lib\elements\satin_column.py", line 839, in do_satin
  File "lib\elements\satin_column.py", line 689, in plot_points_on_rails
  File "lib\elements\satin_column.py", line 424, in flattened_sections
  File "lib\elements\satin_column.py", line 424, in <listcomp>
IndexError: tuple index out of range

![GingerBreadCookie12-30-22-A](https://user-images.githubusercontent.com/92271951/210118953-62050eb8-0227-4664-8518-487e99f25b72.svg)

**Comment by kaalleen** [comment]
Thank you for your error report. I tried your svg in a current developer version and cannot reproduce the error. So I think that we already fixed the error with a nicer error message.

The problem in your svg is, that you applied satin columns to elements, that are not build like a satin column. Select your decorations and outlines and run the params dialog. Disable the satin column (back to a running stitch) and also disable the manual stitch setting. Then apply. Convert your strokes into a satin column with Extensions > Ink/Stitch > Tools: Satin > Convert line to satin column.
